crypto/fipsmodule/evp/internal.h

```diff
@@ -381,16 +381,9 @@ typedef struct {
 void evp_pkey_set_cb_translate(BN_GENCB *cb, EVP_PKEY_CTX *ctx);
 
 #define ED25519_PUBLIC_KEY_OFFSET 32
-
-#ifdef ENABLE_DILITHIUM
 #define FIPS_EVP_PKEY_METHODS 7
```

Comment:
Is this value not changing?

Reply:
We're marking ML-DSA as a `NON_FIPS_EVP_PKEY_METHODS` and yes it went from 3 to 4

Reply:
I thought it was a `FIPS_EVP_PKEY_METHODS` before moving it outside the fipsmodule and now a change is needed in the `FIPS_EVP_PKEY_METHODS` count. 
nvm, thanks!